public/index.html

```diff
@@ -93,25 +93,25 @@
       </div>
 
       <div class="medium-6 columns">
-        <h2>Hovedstyret høst 2016</h2>
+        <h2>Hovedstyret vår 2017</h2>
         <p>Hele hovedstyret kan nås via <a href="mailto:hs@cyb.no">hs@cyb.no</a>.</p>
         <dl class="page-dl">
           <dt>Leder</dt>
           <dd><a href="mailto:leder@cyb.no">Odd-Tørres Lunde</a></dd>
           <dt>Nestleder</dt>
           <dd><a href="mailto:leder@cyb.no">Åsmund Rosendahl</a></dd>
           <dt>Kasserer</dt>
-          <dd><a href="mailto:kasserer@cyb.no">Marius Aasarød</a></dd>
+          <dd><a href="mailto:kasserer@cyb.no">Andreas Nyborg Hansen</a></dd>
           <dt>Kjellermogul</dt>
-          <dd><a href="mailto:kjellermogul@cyb.no">Nikolas Papaioannou</a></dd>
+          <dd><a href="mailto:kjellermogul@cyb.no">Karl Hole Totland</a></dd>
           <dt>Arrangementssjef</dt>
-          <dd><a href="mailto:arrangementssjef@cyb.no">Cindy Madeleine Svendsen Ceesay</a></dd>
+          <dd><a href="mailto:arrangementssjef@cyb.no">Elise Lystad</a></dd>
           <dt>Rekrutteringsansvarlig</dt>
           <dd><a href="mailto:rekrutteringsansvarlig@cyb.no">Synne Baumann</a></dd>
           <dt>Internansvarlig</dt>
           <dd><a href="mailto:internansvarlig@cyb.no">Adrian Helle</a></dd>
-          <dt>Promoteringsansvarlig</dt>
-          <dd><a href="mailto:blestsjef@cyb.no">Silje Merethe Dahl</a></dd>
+          <dt>PR-sjef</dt>
```

Comment:
Vervet er ikke definert i vedtektene så da må man ta det HS har valgt å legge i sakspapirene for valget. Og de valgte å bruke PR-sjef.